LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
 
-compas-dev
+Copyright (c) 2023 COMPAS Association
```

Comment:
haha just out of curiosity, are we expected to update this every year? what does this even mean?

Reply:
Yes, I think we are. There are ever github actions for this: https://github.com/marketplace/actions/update-license-copyright-year-s

Reply:
I'm not entirely certain about the legal implications of claiming copyright every year, I guess it's related to the expiration of copyright.